src/commonMain/kotlin/com/github/quillraven/fleks/component.kt

```diff
@@ -26,6 +26,12 @@ abstract class ComponentType<T> {
     }
 }
 
+/**
+ * Function to create an object for a [ComponentType] of type T.
+ * This is a convenience function for [components][Component] that have more than one [ComponentType].
+ */
+inline fun <reified T> componentTypeOf(): ComponentType<T> = object : ComponentType<T>() {}
```

Comment:
Nice! Will come in handy also here ;)

![image](https://user-images.githubusercontent.com/19785115/220976121-5a615400-22ca-4b70-9e6f-e8d096133979.png)